Source/WebCore/platform/text/TextFlags.h

```diff
@@ -58,20 +58,42 @@ enum class NonCJKGlyphOrientation : uint8_t {
     Upright
 };
 
-enum ExpansionBehaviorFlags {
-    ForbidRightExpansion = 0 << 0,
-    AllowRightExpansion = 1 << 0,
-    ForceRightExpansion = 2 << 0,
-    RightExpansionMask = 3 << 0,
+struct ExpansionBehavior {
+    enum class Behavior : uint8_t {
+        Forbid,
+        Allow,
+        Force
+    };
+
+    ExpansionBehavior()
+        : ExpansionBehavior(Behavior::Forbid, Behavior::Forbid)
```

Comment:
If you use brace-initialization, you might be able to delete this entire constructor. Even if you have to keep the constructor, you can say `ExpansionBehavior() = default;`